shop/src/main/java/ro/msg/learning/shop/repository/StockRepository.java

```diff
@@ -0,0 +1,11 @@
+package ro.msg.learning.shop.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+import ro.msg.learning.shop.domain.Stock;
+
+import java.util.UUID;
+
+@Repository
+public interface StockRepository extends JpaRepository<Stock, UUID> {
```

Comment:
Id of the entity is StockId, so it should be public interface StockRepository extends JpaRepository<Stock, StockId>